src/viam/sdk/components/private/board_client.cpp

```diff
@@ -99,7 +99,12 @@ Board::analog_value BoardClient::read_analog(const std::string& analog_reader_na
     if (!status.ok()) {
         throw GRPCException(status);
     }
-    return response.value();
+    Board::analog_value result = {};
+    result.value = response.value();
+    result.min_range = response.min_range();
+    result.max_range = response.max_range();
+    result.step_size = response.step_size();
+    return result;
```

Comment:
This can also be a one liner:
```
return {response.value(), response.min_range(), response.max_range(), response.step_size()};
```

Reply:
I always get nervous using `{...}` syntax in C++: did I put the items it the right order? What if someone later changes the order (e.g., alphabetizes the fields in the struct)? I can't read this without also pulling up the header file to check against. but I guess here it's pretty benign: the order is the one that makes the most sense, and each value has a name that tells you what it is. Thanks for pushing me to question my assumptions!